src/cases/store/reducers/search-filter.reducer.spec.ts

```diff
@@ -16,16 +25,14 @@ describe('Search Filter Reducer', () => {
       }
     };
 
-    it('should return the default state', () => {
-      const initialState  = undefined;
-      const action = {} as any;
-      const state = fromFilter.reducer(undefined, action);
-
-      expect(state).toBe(initialState);
-    });
+    // it('should return the default state', () => {
```

Comment:
hi franco, a pet hate is leaving unused code commented.  do we need this or can we delete it?